build.go

```diff
@@ -306,9 +306,11 @@ func Build(builder Builder, options ...Option) {
 		}
 	}
 
-	if err := validateSBOMFormats(ctx.Layers.Path, ctx.Buildpack.Info.SBOMFormats); err != nil {
-		config.exitHandler.Error(fmt.Errorf("unable to validate SBOM\n%w", err))
-		return
+	if API != "0.1" && API != "0.2" && API != "0.3" && API != "0.4" && API != "0.5" && API != "0.6" {
```

Comment:
Shall we just check if api == 0.7 for now? we can add semver parsing later.

Reply:
I went with this approach because it is future proof and doesn't need changing in the future. It is very ugly though.

I'm not opposed to just matching on 0.7, but I worry that'll cause something to break in the future.

+100 for better API matching. That should be a target for v2. We should also look at some possible patterns for coding multiple version support in sane ways. IMHO, having If version == xyz blocks all over is hard to maintain. There's got to be something better we can do there for v2.